SumDU/Models/ListData.swift

```diff
@@ -110,15 +110,22 @@ extension ListData {
     return sortedRecords
   }
   
-  /// Function which stores ListData entity
-  static func saveObject(_ listDataObject: ListData?, forKey: String) {
+  // Remove ListData entity from storage
+  static func deleteObject(_ listDataObjects: [ListData], value: ListData, forKey: String) {
```

Comment:
@Gomer0513 what about saveObject function? Isn't it needed anymore?

Reply:
@skliarov I've search through the project and have not found that this function is not used at all. That's why I've decided to remove it.